src/inmanta/server/bootloader.py

```diff
@@ -18,6 +18,7 @@
 from inmanta.server import server
 from inmanta.server.protocol import RESTServer
 from inmanta.server.agentmanager import AgentManager
+from tornado import gen
 
 
 class InmantaBootloader(object):
```

Comment:
document that stop is blocking and startup/shutdown order is hardcoded.

Reply:
What do you mean with "stop is blocking"? stop() is an asynchronous function, so it's non-blocking. Right?

Reply:
I mean that it only returns if the stop is complete (i.e. there is no join)

on second thought, that is pretty default, so no need to document